kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/dhcp/DhcpClientManager.java

```diff
@@ -159,6 +159,7 @@ private static String formCommand(String interfaceName, boolean useLeasesFile, b
             sb.append(interfaceName);
             sb.append(' ');
             if (usePidFile) {
+            	sb.append("-p ");
```

Comment:
May I ask you to apply the Kura formatter here? The line added seems misaligned compared to the others. Thanks